hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOmSnapshotFileSystem.java

```diff
@@ -655,7 +698,6 @@ public void testListStatusOnRoot() throws Exception {
   // based on TestOzoneFileSystem:testListStatusOnLargeDirectory
   public void testListStatusOnLargeDirectory() throws Exception {
     Path root = new Path("/");
-    deleteRootDir(); // cleanup
```

Comment:
Do we not need the cleanup anymore? I believe this is to clean up from the previous test run right?

Reply:
"this is to clean up from the previous test run right" is my understanding as well.